More README.md cleanups regarding retries (#448)
zalando-nakadi/fahrschein: README.md

    nakadiClient.publish(ORDER_CREATED, events);
}    
```

## Publishing Events With Retry

Nakadi publishing API accepts events in batches. It can fail to publish some events from the batch to underlying storage.

> In that case Nakadi publishing API will return error that batch was partially successful, indicated by an `EventPersistenceException`.

Author: ```suggestion
In that case Nakadi publishing API will return error that batch was partially successful, indicated in Fahrschein by an `EventPersistenceException`.
```

For this situation, Fahrschein offers auto-retry mechanisms, allowing clients to make a choice between retrying the entire batch or opting for a partial retry as needed. 

> For added flexibility, NakadiClient allows configuration of various [backoff strategies](https://github.com/zalando-nakadi/fahrschein#backoff-strategies) for handling event publishing retries.

Author: ```suggestion
For added flexibility, various [backoff strategies](https://github.com/zalando-nakadi/fahrschein#backoff-strategies) for delaying the retries exist.
```
